protocols/66e60f/66e60f.py

```diff
@@ -0,0 +1,301 @@
+    describe the parameters which are to be passed in as a docstring below
+    the function (see below).
+
+    def pick_up(pipette):
+        """`pick_up()` will pause the protocol when all tip boxes are out of
+        tips, prompting the user to replace all tip racks. Once tipracks are
+        reset, the protocol will start picking up tips from the first tip
+        box as defined in the slot order when assigning the labware definition
+        for that tip box. `pick_up()` will track tips for both pipettes if
+        applicable.
+
+        :param pipette: The pipette desired to pick up tip
+        as definited earlier in the protocol (e.g. p300, m20).
+        """
+        try:
+            pipette.pick_up_tip()
+        except protocol_api.labware.OutOfTipsError:
+            ctx.pause("Replace empty tip racks")
+            pipette.reset_tipracks()
+            pipette.pick_up_tip()
+
+    '''
+
+    # helper functions
+    '''
+    Define any custom helper functions outside of the pipette scope here, using
+    the convention seen above.
+
+    e.g.
+
+    def remove_supernatant(vol, index):
+        """
+        function description
+
+        :param vol:
+
+        :param index:
+        """
+
+
+    '''
+
+    # reagents
+
+    '''
+    Define where all reagents are on the deck using the labware defined above.
+
+    e.g.
+
+    water = reservoir12.wells()[-1]
+    waste = reservoir.wells()[0]
+    samples = plate.rows()[0][0]
+    dnase = tuberack.wells_by_name()['A4']
+
+    '''
+    water_well = reservoir.wells_by_name()['A1']
+    liquid_waste = reservoir.wells_by_name()['A2']
+
+    # plate, tube rack maps
+
+    '''
+    Define any plate or tube maps here.
+
+    e.g.
+
+    plate_wells_by_row = [well for row in plate.rows() for well in row]
+
+    '''
+
+    # protocol
+
+    '''
+
+    Include header sections as follows for each "section" of your protocol.
+
+    Section can be defined as a step in a bench protocol.
+
+    e.g.
+
+    ctx.comment('\n\nMOVING MASTERMIX TO SAMPLES IN COLUMNS 1-6\n')
+
+    for .... in ...:
+        ...
+        ...
+
+    ctx.comment('\n\nRUNNING THERMOCYCLER PROFILE\n')
+
+    ...
+    ...
+    ...
+
+
+    '''
+
+    ctx.comment("\nTransferring water to target plate\n")
+    p300.pick_up_tip()
+    for well in target_plate.wells():
+        if p300.current_volume < initial_water_volume:
+            p300.aspirate(200-p300.current_volume, water_well)
+        p300.dispense(initial_water_volume, well)
+    p300.blow_out(water_well)
+    p300.return_tip()
+    p300.reset_tipracks()
+
+    # Transfering DNA samples to target
+    ctx.comment("\nTransferring DNA sample to target plate\n")
+    for line in data:
+        well = line[0]
+        description = line[1]
+        concentration = line[2]
+        volume = float(line[3])
+
+        ctx.comment("Normalizing sample {} with concentration {}"
+                    .format(description, concentration))
+
+        if volume <= 20:
```

Comment:
Could be a function or an if statement
if vol <= 20: pip = p20, else: pip = p300

Reply:
pip = p20 if vol <= 20 else p300